plugin/braceless.vim

```diff
@@ -54,16 +54,12 @@ function! s:enable(...)
 
   if !empty(g:braceless#key#jump_prev)
     execute 'map <buffer> ['.g:braceless#key#jump_prev.' <Plug>(braceless-jump-prev-n)'
-    execute 'map <buffer> g'.g:braceless#key#jump_prev.' <Plug>(braceless-jump-prev-n-indent)'
```

Comment:
Could you turn this into an option that's disabled by default instead?  There might be people who want to retain this feature and I don't want to blindside anyone without an option to get it back.